src/options/Keybindings/default.json

```diff
@@ -654,6 +654,123 @@
+        "scrollToLeft": [],
+        "scrollToRight": [],
+        "goBack": [[{ "code": "KeyH", "key": "H", "shiftKey": true }]],
+        "goForward": [[{ "code": "KeyL", "key": "L", "shiftKey": true }]],
+        "nextPage": [],
+        "previousPage": [],
+        "goUp": [
+          [{ "code": "KeyG", "key": "g" }, { "code": "KeyU", "key": "u" }]
+        ],
+        "goToRoot": [
+          [
+            { "code": "KeyG", "key": "g" },
+            { "code": "KeyU", "key": "U", "shiftKey": true }
+          ]
+        ],
+        "previousTab": [
+          [{ "code": "KeyJ", "key": "J", "shiftKey": true }]
+        ],
+        "nextTab": [
+          [{ "code": "KeyK", "key": "K", "shiftKey": true }]
+        ],
+        "firstTab": [
+          [{ "code": "KeyG", "key": "g" }, { "code": "Digit0", "key": "0" }]
+        ],
+        "lastTab": [
+          [
+            { "code": "KeyG", "key": "g" },
+            { "code": "Digit4", "key": "$", "shiftKey": true }
+          ]
+        ],
+        "moveTabLeft": [
+          [
+            { "code": "KeyG", "key": "g"},
+            { "code": "KeyJ", "key": "J", "shiftKey": true }
+          ]
+        ],
+        "moveTabRight": [
+          [
+            { "code": "KeyG", "key": "g"},
+            { "code": "KeyK", "key": "K", "shiftKey": true }
+          ]
+        ],
+        "moveTabFirst": [],
+        "moveTabLast": [],
+        "closeTab": [[{ "code": "KeyX", "key": "x" }]],
+        "closeOtherTabs": [[{ "code": "KeyG", "key": "g" }, {"code": "KeyX", "key": "x"}, {"code": "KeyA", "key": "a"} ]],
+        "closeRightTabs": [[{ "code": "KeyG", "key": "g" }, {"code": "KeyX", "key": "x"}, {"code": "Digit4", "key": "$", "shiftKey": true } ]],
+        "closeLeftTabs": [[{ "code": "KeyG", "key": "g" }, {"code": "KeyX", "key": "x"}, {"code": "Digit0", "key": "0" } ]],
+        "newTab": [[{ "code": "KeyT", "key": "t" }]],
+        "restoreTab": [[{ "code": "KeyX", "key": "X", "shiftKey": true }]],
+        "duplicateTab": [
+          [{ "code": "KeyY", "key": "y" }, { "code": "KeyT", "key": "t" }]
+        ],
+        "newWindow": [[{ "code": "KeyW", "key": "w" }]],
+        "newIncognitoWindow": [
+          [{ "code": "KeyW", "key": "W", "shiftKey": true }]
+        ],
+        "switchWindow": [],
+        "zoomIn": [],
+        "zoomOut": [],
+        "zoomReset": [],
+        "refreshTab": [[{ "code": "KeyR", "key": "r" }]],
+        "refreshAllTab": [[{ "code": "KeyA", "key": "a" }, { "code": "KeyR", "key": "r" } ]],
+        "hardRefreshTab": [[{ "code": "KeyR", "key": "R", "shiftKey": true }]],
+        "toggleMuteTab": [],
+        "toggleMuteAllTabs": [],
+        "togglePinTab": [[{ "code": "KeyG", "key": "g" },{ "code": "KeyP", "key": "p"}]],
+        "passOneKey": [[{ "code": "KeyI", "key":"I", "shiftKey": true }]],
+        "passAllKeys": [{ "code": "KeyI", "key":"i"}],
+        "exitPassMode": [],
+        "exitPassMode": [[{ "code": "Escape", "key":"Esc"}]],
```

Comment:
You seem to have specified `exitPassMode` twice here.

Reply:
yup, missed that